ignite/pkg/xgit/xgit.go

```diff
@@ -18,23 +18,44 @@ var (
 	}
 )
 
+// InitAndCommit creates a git repo in path if path isn't already inside a git
+// repository, then commits path content.
 func InitAndCommit(path string) error {
```

Comment:
Do you think we can test this function?

Would just be tracked in a new issue.  Not part of this scope.

Reply:
sure it's possible, I've already created git repositories on the fly during the load plugin tests.

Reply:
Can we create an issue for testing this in the future?

Reply:
sure, done there https://github.com/ignite/cli/issues/3125